woocommerce/woocommerce-ios
# Products Preview: Add loginURL to Site

Part of https://github.com/woocommerce/woocommerce-ios/issues/7911

## Description

This PR adds `loginURL` property to:
- Networking: `me/sites` query options
- Networking: Site object mapping
- Storage: Site object

and updates related tests for:
- Networking mapping
- Storage model version
- UI tests mocks

We need `loginURL` for web auth flow, like it already [used](https://github.com/wordpress-mobile/WordPress-iOS/blob/3189c40cd115557d70bc7fa5c08385b331b76fe4/WordPress/Classes/Utility/Networking/RequestAuthenticator.swift#L209) in WP app.

## Testing

1. Launch the app, logout if needed.
2. Login back, confirm that stores list is loaded correctly.
3. Select a store, confirm it loads correctly.

---
